.circleci/config.yml

```
@@ -25,21 +25,48 @@ commands:
           command: docker-compose up -d
 
 jobs:
-  build_and_test:
+  build:
     working_directory: ~/repo
     docker:
       - image: cimg/node:18.16.0
       - image: cimg/postgres:14.1
+    resource_class: medium
     steps:
       - checkout
+      - restore_cache:
+          key: v1-deps-{{ checksum "package-lock.json" }}
```

I recommend reading `v1` from environment variables - it will give you control over cache without deploying changes in the code, e.g.:
```suggestion
          key: {{ .Environment.CACHE_VERSION }}-deps-{{ checksum "package-lock.json" }}
```